_storage/amazon-s3-deploy.md

```diff
@@ -0,0 +1,466 @@
+---
+layout: storage
+permalink: /:collection/:path.html
+---
+# Configure a hub on Amazon EC2
+{:.no_toc}
+
+This teaches you how to run a Gaia hub on Amazon EC2. Amazon EC2 is an affordable and convenient cloud server provider. This example uses Amazon EC2 together with an EB3 instance for file storage.
+
+* TOC
+{:toc}
+
+<div class="uk-card uk-card-default uk-card-body">
+<h5>Is this tutorial for you?</h5>
+
+<p>This documentation is appropriate for advanced power users who are familiar with command line tools, <code>ssh</code>, and basic editing configuration files.</p>
+
+<p>If you are planning on running an <emphasis>open-membership hub</emphasis> or an <emphasis>application-specific hub</emphasis>, see <a href="hub-operation.html">the section on Hub Operation</a>.</p>
+
+</div>
+
+## Prerequisites you need
+
+This procedure uses Amazon AWS to choose and configure an Amazon Machine Image
+(AMI) running a Gaia service. For this reason, you should have an AWS account
+on the <a href="https://aws.amazon.com/free/" target="\_blank">Amazon AWS free
+tier</a>, personal account, or corporate account.  These instructions assume you
+are using a free tier account.
+
+These instructions assume you have already created a free <a
+href="https://www.freenom.com" target="\_blank">domain through the freenom
+service</a>. If you have another domain, you can use that instead.
+
+Finally, setting up the SSL certificates on your EC2 instance requires you to use the terminal command line on your workstation. Make sure you have the `watch` command installed using the `which` command.
+
+```
+$ which watch
+/usr/local/bin/watch
+```
+
+If `watch` is not located, install it on your workstation.
+
+## Task 1: Launch an EC2 instance
+
+1. Visit the <a href="https://aws.amazon.com/free/" target="\_blank">AWS Free Tier page</a> and choose **Sign in to the Console**.
+
+   ![](/storage/images/aws-console.png)
+
+2. Make sure your region is set to `us-west-2` which is **Oregon**.
+
+   ![](/storage/images/us-west-2.png)
+
+3. Choose **Launch a virtual machine**.
```

Comment:
My AWS UI seems to say "Launch Instance"s with a blue button instead of this

Reply:
It says this because you went to the **EC2** dashboard rather than the AWS dashboard.